app/models.py

```diff
@@ -2142,3 +2148,56 @@ def serialize(self):
             "created_at": created_at_in_bst.strftime("%Y-%m-%d %H:%M:%S"),
         }
         return contact_list
+
+
+class BroadcastMessage(db.Model):
+    __tablename__ = 'broadcast_message'
+    __table_args__ = (
+        db.ForeignKeyConstraint(
+            ['template_id', 'template_version'],
+            ['templates_history.id', 'templates_history.version'],
+        ),
+        {}
+    )
+
+    STATES = [
+        'draft',
+        'pending-approval',
+        'rejected',  # rejected before leaving notify
+        'broadcasting',  # approved, now live
```

Comment:
We've decided to go from pending-approval to broadcasting... do we need a state to represent the time from clicking the approve button to the network accepting the broadcast and the broadcast has started?

Reply:
hmmm, i don't think there's a need for MVP. trying to think of something that's almost similar, we don't, for example, have a state for "antivirus has completed but validation hasn't happened yet" for precompiled letters - we get the `process-antivirus-passed` task and immediately begin validation.

Reply:
No we don't have pending-validation, but now that it happens as part of a new task we may want to introduce it. However, we haven't seen a need to do that yet. 

If we use `start_at` as a send broadcast message start at that time, then we may want a state to represent that. Not sure so probably not needed yet